src/renderless/CoreForm.vue

```diff
@@ -284,6 +284,9 @@ export default {
             return section.slot || section.fields
                 .some(field => !field.meta.hidden);
         },
+        isSectionVisible(section) {
```

Comment:
1. why the '?'

2. I believe that if we show sections based on visible fields too, we won't solve the problem we're trying to solve with this PR. If somebody will want to hide a section will do it directly, not by hiding the fields.